packages/constants/src/feature-toggles.ts

```diff
@@ -18,10 +18,13 @@
  */
 
 export const CUSTOM_VIEWS = 'enableCustomViews';
+export const ENABLE_WORKSPACES_UI = 'enableWorkspacesUi';
 
 export const FLAGS = {};
 
 // Long-lived feature flags, defined in the MC API.
 export const DEFAULT_FLAGS = {
   [CUSTOM_VIEWS]: { value: true },
+  // TODO: set to false before making the PR
+  [ENABLE_WORKSPACES_UI]: { value: true },
```

Comment:
Still working on this, hence the draft mode. Will set to false when the time comes